includes/admin/metaboxes/settings.php

```diff
@@ -663,6 +663,27 @@ class="simcal-field simcal-field-text simcal-field-small"
 				</td>
 			</tr>
 			<tr class="simcal-panel-field">
+				<th><label for="_calendar_datetime_separator_spacing"><?php _e( 'Preserve Separator Spacing', 'google-calendar-events' ); ?></label></th>
+				<td>
+					<?php
+
```

Comment:
Can we remove this blank line after the opening `<?php` tag? Or is this a WPCS thing?